tests/test_data/brokenovenbaking.com/brokenovenbaking_1.json

```diff
@@ -0,0 +1,67 @@
+{
+  "author": "Kayla Burton",
+  "canonical_url": "https://brokenovenbaking.com/easy-homemade-strawberry-muffins/",
+  "site_name": "Broken Oven Baking",
+  "host": "brokenovenbaking.com",
+  "language": "en-US",
+  "title": "Easy Homemade Strawberry Muffins",
+  "ingredients": [
+    "2¼ cups all-purpose flour (divided (weighed in grams for best results))",
+    "2 teaspoons baking powder",
+    "½ teaspoon salt",
+    "½ cup vegetable oil (or other neutral oil)",
+    "1 cup granulated sugar",
+    "2 large eggs (room temperature)",
+    "1½ teaspoons vanilla extract",
+    "¼ teaspoon almond extract (optional)",
+    "½ cup Lifeway Organic Strawberry Whole Milk Kefir (room temperature)",
+    "2 cups chopped strawberries (divided)"
+  ],
+  "instructions_list": [
+    "Preheat oven to 375°F and fill a muffin pan with liners or grease with non-stick spray.",
+    "In a medium bowl, whisk together 2 cups of flour, the baking powder, and salt.",
+    "In a large bowl, whisk together the vegetable oil, sugar, eggs, vanilla extract, almond extract (if using), and strawberry kefir.",
+    "Chop and measure the strawberries. Pat them dry if they're super juicy.Set aside ½ cup of the chopped berries. Coat 1½ cups of them in ¼ cup of flour.",
+    "Stir the dry ingredients into the wet ingredients until just a few flour streaks are visible.",
+    "Fold the flour-coated strawberries into the batter just until combined.",
+    "Fill the muffin liners about ¾ full with batter. Top with the remaining ½ cup of strawberries and optional coarse sugar.",
+    "Bake on the middle rack until a toothpick comes out of the center clean (about 20-25 minutes).",
+    "Once the muffins are cool enough to touch, transfer them to a wire cooling rack."
+  ],
+  "category": "Breakfast,Brunch,Dessert,Snack",
+  "yields": "12 servings",
+  "description": "Enjoy homemade strawberry muffins for breakfast in just 30 minutes! This recipe is easy to make with real fresh or frozen strawberries.",
+  "total_time": 35,
+  "cook_time": 20,
+  "prep_time": 15,
+  "cuisine": "American",
+  "ratings": 5.0,
+  "ratings_count": 2,
+  "equipment": [
+    "Muffin panfilled with baking cups or lightly greased",
```

Comment:
Something a bit odd going on with the text parsing here - perhaps an HTML decoding problem (`panfilled` in particular)?